### PR TITLE
Remove deprecated options

### PR DESCRIPTION
**Pull request by MadeByMike**
Noticed this in my travels and thought I'd do a quick PR.

### CODE REVIEW

File: .changeset/famous-numbers-happen/changes.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{ "releases": [{ "name": "@keystone-alpha/app-graphql", "type": "patch" }], "dependents": [] }
```
Comment (jesstelford) [inline .changeset/famous-numbers-happen/changes.json]: I don't think this file should exist with the latest changesets. Make sure your branch is from the HEAD of `master` and that you've reinstalled all your dependencies before rerunning the `changeset` command.

Comment (mitchellhamilton) [reply .changeset/famous-numbers-happen/changes.json]: Changesets 2 can handle changesets in this format so it's totally fine to leave this btw.

File: website/.changeset/config.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
+/*
```
Comment (mitchellhamilton) [inline website/.changeset/config.js]: This file and `website/.changeset/README.md` shouldn't exist

Comment (MadeByMike) [reply website/.changeset/config.js]: Ahh I must have run change set in the website dir :(

**Pull request merged**